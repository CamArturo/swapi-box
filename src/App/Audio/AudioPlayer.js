import React from 'react';
import jediSteps from '../assets/23-The-Jedi-Steps-and-Finale.mp3';
// import farewell from '/Audio/22-Farewell-and-the-Trip.mp3';

const AudioPlayer = props => {
  return (
    <div>
      <audio className="mainAudio" autoPlay loop>
        {/*<source src={jediSteps}/>*/}
      </audio>
    </div>
  );
};

export default AudioPlayer;
