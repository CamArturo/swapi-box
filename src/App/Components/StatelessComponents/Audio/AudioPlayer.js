import React from 'react';
import jediSteps from '../../../assets/23-The-Jedi-Steps-and-Finale.mp3';

const AudioPlayer = () => {
  return (
    <div>
      {/*<audio src={jediSteps} className="mainAudio" autoPlay loop preload>*/}
      {/*</audio>*/}
    </div>
  );
};

export default AudioPlayer;