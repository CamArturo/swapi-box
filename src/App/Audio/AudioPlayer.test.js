import React from 'react';
import {shallow} from 'enzyme';
import AudioPlayer from "./AudioPlayer";

it('renders without crashing', () => {
  const wrapper = shallow(<AudioPlayer/>);

  expect(wrapper).toMatchSnapshot();
});
