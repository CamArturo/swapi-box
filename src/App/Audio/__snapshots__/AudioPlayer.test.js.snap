// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <AudioPlayer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <audio
        autoPlay={true}
        className="mainAudio"
        loop={true}
>
        <source
                src="23-The-Jedi-Steps-and-Finale.mp3"
        />
</audio>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "autoPlay": true,
        "children": <source
          src="23-The-Jedi-Steps-and-Finale.mp3"
/>,
        "className": "mainAudio",
        "loop": true,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "src": "23-The-Jedi-Steps-and-Finale.mp3",
        },
        "ref": null,
        "rendered": null,
        "type": "source",
      },
      "type": "audio",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <audio
          autoPlay={true}
          className="mainAudio"
          loop={true}
>
          <source
                    src="23-The-Jedi-Steps-and-Finale.mp3"
          />
</audio>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "autoPlay": true,
          "children": <source
            src="23-The-Jedi-Steps-and-Finale.mp3"
/>,
          "className": "mainAudio",
          "loop": true,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "src": "23-The-Jedi-Steps-and-Finale.mp3",
          },
          "ref": null,
          "rendered": null,
          "type": "source",
        },
        "type": "audio",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
